modules/admin/client/components/search.api.js

```diff
@@ -0,0 +1,10 @@
+export async function searchUsers(query) {
+  const response = await fetch(`/api/users?search=${query}`, {
```

Comment:
Let's make sure `fetch` is polyfilled in Babel to not break Internet Explorer and some mobile browsers. Lemme know if you need help with that!

Reply:
More info [here](https://github.com/Trustroots/trustroots/pull/790#issuecomment-437868403). This will be needed in parallel in #827 and in Search users.

Reply:
How about we land [Axios](https://www.npmjs.com/package/axios) and this API helper in one PR before this and user search PRs land?

Reply:
IE use is still ~1%, but none of those will (should) be admins, so let's break MSIE! 🤣